common.gradle

```
@@ -198,6 +198,7 @@ processResources {
 				"version": fullModVersion,
 				"minecraft_dependency": project.minecraft_dependency,
 				"malilib_dependency": project.malilib_dependency,
+				"litematica_conflict": project.getProperties().getOrDefault("litematica_conflict", "<0.0.0"),
```

litematica for mc < 1.17 are versioned with `0.0.0-xxx`, which will hit this predicate

```suggestion
				"litematica_conflict": project.getProperties().getOrDefault("litematica_conflict", "<0.0.0-"),
```